admin/fluent-bit/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fluent-bit
+PKG_VERSION:=3.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/fluent/fluent-bit/archive/refs/tags/
+PKG_HASH:=7a49e110cf3050b6c29c911063494b8081f3c743274d1d95e52562d0476ba1eb
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/fluent-bit
+	CATEGORY:=Utilities
+	DEPENDS:= +libyaml +libopenssl +libcurl +libatomic +musl-fts +flex +bison
+	TITLE:=Fluent-Bit
+	URL:=https://fluentbit.io/
+endef
+
+define Package/fluent-bit/description
+	Fluent Bit is a super fast, lightweight, and highly scalable logging and metrics processor and forwarder.
+endef
+
+# General options
+TARGET_LDFLAGS +=-lfts -latomic
+
+CMAKE_OPTIONS+= \
+	-DFLB_RELEASE=Yes \
+	-DEXCLUDE_FROM_ALL=true \
+	-DFLB_EXAMPLES=No \
+	-DFLB_CHUNK_TRACE=No \
+	-DFLB_BACKTRACE=No \
+	-DFLB_WASM=No \
+	-DFLB_LUAJIT=No
+
+define Package/fluent-bit/install
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/etc/fluent-bit
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/fluent-bit $(1)/usr/sbin/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/conf/parsers.conf $(1)/etc/fluent-bit/parsers.conf
+endef
```

Comment:
We can order it a little bit more.

First folder, then file.

Also, the config file should be INSTALL_CONF and shouldnt it be marked as conffile to prevent to be overwritten by update?

Reply:
done